packages/astro-uxds/_content/components/clock.md

```diff
@@ -13,16 +13,16 @@ theme: true
 
 # Clock
 
-Clock shows the current time and optionally, Julian date, AOS, and LOS timers. It will typically be positioned on the Global Status Bar.
+Clock shows the current time and, optionally, Julian date, AOS, and LOS timers. It will typically be positioned on the Global Status Bar.
 
 ## Appearance and Behavior
 
-The clock is not an interactive component. Date and Time are always present. AOS, and LOS are optional. The time is UTC by default, but can be configured for any time zone.
+The clock is not an interactive component. Date and Time are always present. AOS and LOS are optional. The time is UTC by default but can be configured for any time zone.
 
-The optional date feature uses the Julian day number format. Julian day number format requires 3 digits with placeholder 0s for the hundreds and tens place.
+The optional date feature uses the Julian day number format. Julian day number format requires 3 digits with placeholder 0s for the hundreds and tens places.
 
-All digits should be displayed using the system mono font, system font or Roboto with [tabular numbers](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric) enabled. Mono space font and [Tabular number](https://www.fonts.com/content/learning/fontology/level-3/numbers/proportional-vs-tabular-figures) geometry ensure that the display will not jitter as the numbers change.
+All digits should be displayed using the system mono font, system font, or Roboto with [tabular numbers](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric) enabled. Mono space font and [Tabular number](https://www.fonts.com/content/learning/fontology/level-3/numbers/proportional-vs-tabular-figures) geometry ensure that the display will not jitter as the numbers change.
```

Comment:
@micahjones13 Shouldn't we be giving Roboto Mono as an example of acceptable font options as well if we're not doing a breaking change?

Reply:
This was straight from Duncan so I'm not entirely sure. @dmcalester what do you think?